cypress/integration/step_definitions/backend/subscription.cy.js

```diff
@@ -0,0 +1,35 @@
+import { And, Given, Then, When } from "cypress-cucumber-preprocessor/steps";
+
+let response = null
```

Comment:
In js, you don't need to initialize value to as `null ` you can leave it as `let response`

Reply:
Thank you for the review